build/Build.Pack.cs

```diff
@@ -23,6 +23,9 @@ partial class Build
     //We don't sign linux packages when building locally
     readonly bool SignLinuxPackages = !IsLocalBuild;
 
+    [Parameter(Name = "DockerBuilder")] string? DockerBuildxBuilder;
+    [Parameter(Name = "DockerPlatform")] string DockerPlatform = "linux/arm64,linux/amd64";
```

Comment:
Is it worth adding some descriptions for these? If you do `./build.[sh/ps1] --help` you get a list of parameters and the description could help someone with what value they need without looking at the code. (I only found this out yesterday 😅 )

<img width="799" alt="Screenshot 2024-02-22 at 13 50 11" src="https://github.com/OctopusDeploy/OctopusTentacle/assets/97430840/679bc315-9b28-4e97-9e01-4bfbef43bc7d">

Reply:
ok yeah, I didn't know that was a thing. Will do

Reply:
Updated in ba48c6b9